contents/docs/self-host/deploy/aws.mdx

```diff
@@ -63,6 +63,12 @@ ingress-nginx:
         service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
 ```
 
+If using an AWS application load balancer, you do not need to add the above
```

Comment:
Does this change only apply to AWS ALB?

Reply:
Hmmm maybe that's not accurate. I think probably more accurate to say if you're terminating TLS at the LB 🤔